server/routes/utils/voteModelUpdates.js

```diff
@@ -26,7 +26,7 @@ export async function registerVote(pollId, userId, option) {
             }
         );
 
-        return { status: 200, pollId: pollId, option: option };
```

Comment:
I took out the status code as it was redundant here and it just gets added in the route.

Reply:
If there is an error it will be caught in the catch statement.

Reply:
Thank you! I get status code 200 in response.status anyway.